src/Utils/Types/VendorState.php

```diff
@@ -7,4 +7,5 @@ class VendorState
     const INSERT = 'insert';
     const UPDATE = 'update';
     const DELETE = 'delete';
+    const DELETE_AND_UPDATE = 'DeleteAndUpdate';
```

Comment:
Why is this camel case, when the others are lowercase?

Reply:
Changed first word to lower case.